debian/changelog

```diff
@@ -1,3 +1,10 @@
+tiqit (1.0.14-1) trusty; urgency=low
+
+  * Fix issues with OIDC redirects causing newlines in summaries and RNE
+    fields and improve the same handling for enclosures.
```

Comment:
Presumably tiqit isn't aware of the concept of an RNE field (that's specific to a particular tiqit client), so this description should refer to something more generic?